pallets/nft/src/lib.rs

```diff
@@ -158,6 +158,7 @@ pub mod pallet {
 		/// - depends on the arguments
 		/// # </weight>
 		#[pallet::weight(<T as Config>::WeightInfo::validate_mint())]
+		#[pallet::call_index(0)]
```

Comment:
It's a new Substrate thing. It requires every extrinsic to have an explicit index. This index is used - probably amongst other things - to encode/decode calls.

Reply:
Note that all of these things could be done without `#[pallet::call_index(_)]`, but then it would be implicit. So the idea is to make extrinsic indexing explicit and hopefully harder / more obvious to mess up.

Reply:
Oh, I understand now. Thanks, both for your detailed answer! 🙌🏻